releaf-content/spec/features/nodes_spec.rb

```diff
@@ -128,7 +128,8 @@
           click_button "Copy"
         end
 
-        expect(page).to have_css('.dialog .form-error-box', text: "Source or descendant node can't be parent of new node")
+        error_text = 'Node with id 3 has error "source or descendant node can\'t be parent of new node"'
+        expect(page).to have_css('.dialog .form-error-box', text: error_text)
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.